### PR TITLE
Electrum servers list updates

### PR DESCRIPTION
**Pull request by nkuba**
### Blockstream's Servers Keep Alive Interval (ba416cedb989b0eebc8768ec7fc56de662647803)
Blockstream's servers timeout session after 60 seconds of inactivity which
is much shorter than the expected 600 seconds (see: [Electrum Protocol Docs](https://electrumx-spesmilo.readthedocs.io/en/latest/environment.html#envvar-SESSION_TIMEOUT)).
To workaround connection drops and logs pollution with warnings, we reduce the
`KeepAliveInterval` for Blockstream's servers to less than 60 seconds.
 
### Integration Tests for Embedded Electrum Servers (a288918fc657a779ec09b0b8b14a4bd1a80c4468)
We add an integration test to check the client can establish a connection to the embedded servers.

To run it execute:
```
go test -v -tags=integration ./config -run TestEmbeddedElectrumServerConnect_Integration
```

### Add Electrum Servers (e677fc312c87febe1080d69db3c11e5883dd6847)

We add more servers to the Electrum Servers list.

### CODE REVIEW

File: config/_electrum_urls/mainnet [hunk 1]
```diff
@@ -1 +1,3 @@
-ssl://electrum.blockstream.info:50002
\ No newline at end of file
+wss://electrumx-server.tbtc.network:8443
+wss://electrum.boar.network:2083
+ssl://electrum.blockstream.info:50002
```
Comment (pdyraga) [inline config/_electrum_urls/mainnet:3]: P2P set up their Electrum server. It is available at:
wss://bitcoin.threshold.p2p.org:50004
ssl://bitcoin.threshold.p2p.org:50002

Comment (nkuba) [reply config/_electrum_urls/mainnet:3]: It wasn't available the last time I checked. I'll test it again.

File: config/electrum.go [hunk 2]
```diff
@@ -63,11 +70,20 @@ func (c *Config) resolveElectrum() error {
...
 	c.Bitcoin.Electrum.URL = selectedURL
 
+	// Blockstream's servers timeout session after 60 seconds of inactivity which
+	// is much shorter than expected 600 seconds. To workaround connection drops
+	// and logs pollution with warning we reduce the KeepAliveInterval for the
+	// Blockstream's servers to less than 60 seconds.
+	if c.Bitcoin.Electrum.KeepAliveInterval == 0 &&
+		strings.Contains(selectedURL, "electrum.blockstream.info") {
+		c.Bitcoin.Electrum.KeepAliveInterval = blockstreamKeepAliveInterval
+	}
```
Comment (pdyraga) [inline config/electrum.go:86]: This is good for now but ultimately, I'd rather replace `config/_electrum_urls/mainnet` file with `config/electrum/mainnet` directory and separate config files for each electrum server.

File: config/_electrum_urls/mainnet [hunk 3]
```diff
@@ -1 +1,4 @@
-ssl://electrum.blockstream.info:50002
\ No newline at end of file
+wss://electrumx-server.tbtc.network:8443
+wss://electrum.boar.network:2083
+wss://bitcoin.threshold.p2p.org:50004
+ssl://electrum.blockstream.info:50002
```
Comment (pdyraga) [inline config/_electrum_urls/mainnet:4]: Should we include all those servers in tests executed in `/pkg/bitcoin/electrum/electrum_integration_test.go`? 

Checking if the connection works is fine but checking if they can execute e.g. `GetLatestBlockHeight` is even better.

Comment (nkuba) [reply config/_electrum_urls/mainnet:4]: Yes, we should have a way to validate the servers, it's tracked in https://github.com/keep-network/keep-core/issues/3594.

**Comment by pdyraga** [comment]
Tested locally and works as advertised. All mainnet servers are randomly selected and I no longer see warnings for Blockstream server. To test locally, I had to add a small change: #3596 so that I can see to which Electrum server I am actually connected.

**Review by pdyraga — APPROVED** [review]

**Pull request merged**